components/Navbar.tsx

```diff
@@ -0,0 +1,278 @@
+import Link from "next/link"
+
+interface RouteItem {
+  title: string
+  href: string
+}
+
+interface NavigationProps {
+  routes: RouteItem[]
+  parentTitle: string
+  parentHref: string
+}
+
+const navigation = [
+  {
+    title: "Services",
+    href: "/services",
+    routes: [
+      { title: "Classroom", href: "/services/classroom" },
+      { title: "Computing", href: "/services/computing" },
+      {
+        title: "Campus File Storage and Transfer",
+        href: "/services/file-storage-and-transfer",
+      },
+      { title: "Rates", href: "/services/rates" },
+      { title: "Visualization Systems", href: "/services/visualization" },
+      { title: "Consulting", href: "/services/consulting" },
+    ],
+  },
+  {
+    title: "Documentation",
+    href: "/documentation",
+    routes: [
+      { title: "Oscar", href: "https://docs.ccv.brown.edu" },
+      {
+        title: "Stronghold",
+        href: "/services/computing#data-risk-level-3-computing-(stronghold)",
+      },
+      { title: "Globus", href: "/services/file-storage-and-transfer#globus" },
+      { title: "Jupyter Hub", href: "/services/classroom#jupyterhub" },
+    ],
+  },
+  {
+    title: "Our Work",
+    href: "/our-work",
+    routes: [
+      { title: "Collaborations", href: "#" },
+      {
+        title: "Software",
+        href: "/our-work/software",
+      },
+      {
+        title: "Workshops and Talks",
+        href: "/our-work/workshops-and-talks",
+      },
+      {
+        title: "Publications",
+        href: "https://publications.ccv.brown.edu",
+      },
+    ],
+  },
+  {
+    title: "Help",
+    href: "/help",
+    routes: [
+      { title: "Submit a Ticket", href: "mailto:support@ccv.brown.edu" },
+      {
+        title: "Discourse",
+        href: "https://ask.cyberinfrastructure.org/c/brown-research-computing/37",
+      },
+      {
+        title: "Slack",
+        href: "https://ccv-share.slack.com/signup#/",
+      },
+      {
+        title: "Office Hours",
+        href: "https://events.brown.edu/ccv/month",
+      },
+    ],
+  },
+  {
+    title: "About",
+    href: "/about",
+    routes: [
+      { title: "Mission", href: "/about#mission" },
+      {
+        title: "Office of Information Technology",
+        href: "/about#office-of-information-technology",
+      },
+      {
+        title: "Our Teams",
+        href: "/about#our-teams",
+      },
+      {
+        title: "People",
+        href: "/about#people",
+      },
+      {
+        title: "Opportunities",
+        href: "/about#opportunities",
+      },
+      {
+        title: "Events",
+        href: "https://events.brown.edu/ccv/month",
+      },
+      {
+        title: "Facilities Statement",
+        href: "/about#facilities-statement",
+      },
+      {
+        title: "Diversity Statement",
+        href: "/about#diversity-statement",
+      },
+    ],
+  },
+  {
+    title: "Blog",
+    href: "/blog",
+    routes: [],
+  },
+]
```

Comment:
This is a very Me thing, but I almost feel like I'd want this in a different file? Like in `components/Navbar` have `Navbar.tsx` and `NavItems.ts`. But given that your components folder is flat, I'm not sure it's worth it.... 🤔 

I do like this pattern! Im just trying to think if there's a better way to organize it

Reply:
Given that the footer needs access to this as well, I am in favor of a separate file. The components folder is (eventually) not going to be so flat once everything is written, anyway.